HeavyIonsAnalysis/JetAnalysis/python/FullJetSequence_nominalPbPb.py

```diff
@@ -5,6 +5,8 @@
 from RecoJets.Configuration.GenJetParticles_cff import *
 from Configuration.StandardSequences.ReconstructionHeavyIons_cff import voronoiBackgroundPF, voronoiBackgroundCalo
 
+ak4HiGenJets.doAreaFastjet = cms.bool(True)
```

Comment:
Ah, yes, they are all true [1]. I was playing a bit with it since refarea is always zero for some reason. But that line doesn't solve it so I can remove it. We're mostly interested in getting the area of the reconstructed jet. But if we can get it also for the gen jets that would be great.

[1] https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/RecoHI/HiJetAlgos/python/HiGenJets_cff.py